sympy/core/numbers.py

```diff
@@ -525,6 +534,7 @@ def mod_inverse(a, m):
 
     References
     ==========
+
     - https://en.wikipedia.org/wiki/Modular_multiplicative_inverse
```

Comment:
Can you enumerate the references like `.. [1]`?

Reply:
Sure. Done.